@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/canvas/fn_canvasDrawLine.sqf

```diff
@@ -16,28 +16,28 @@ params
 private ["_ctr", "_dir", "_length"];
 
 _canvas params ["_canvasCtrl", "_canvasCenter", "_canvasVect"];
+private _ctr = (_from vectorAdd _to) vectorMultiply 0.5;
 
 if (_posMode == MPD_POSMODE_WORLD) then {
-    private _ctrWorld = (_from vectorAdd _to) vectorMultiply 0.5;
-    private _theta = [_heli, _heliPos # 0, _heliPos # 1,  _ctrWorld # 0, _ctrWorld # 1, _heading] call fza_fnc_relativeDirection;
-    private _r = _heli distance2d _ctrWorld;
+    private _theta = [_heli, _heliPos # 0, _heliPos # 1,  _ctr # 0, _ctr # 1, _heading] call fza_fnc_relativeDirection;
+    private _r = _heli distance2d _ctr;
     _ctr =
         [ _heliCtr # 0 + sin _theta * (_r * _scale)
-        , _heliCtr # 1 + sin _theta * (_r * _scale)];
+        , _heliCtr # 1 + -cos _theta * (_r * _scale)];
 
     _length = (_from distance2D _to) * _scale;
 
     _dir = (_from getDir _to) - _heading;
 } else {
-    _ctr = (_from vectorAdd _to) vectorMultiply 0.5;
     _length = _from distance2D _to;
     _dir = (_from getDir _to);
 };
 _ctr = [_canvas, _ctr] call fza_ihadss_fnc_canvasAdjust;
+
 _canvasCtrl drawRectangle
     [ _ctr
-    , _length/2*vectorMagnitude _canvasVect
-    , _width/2*vectorMagnitude _canvasVect
+    , (_width/2.8)*vectorMagnitude _canvasVect
+    , (_length/2.8)*vectorMagnitude _canvasVect
```

Comment:
go ask the arma god, 
once i fixed the drawing system the lines did not stretch to meet eachother

Reply:
I see why now. I should have had `vectorMagnitude` divided by `sqrt 2` - which happens to be ~1.41. Can we do that instead? 

Alternatively, using `_canvasVect#0` and `_canvasVect#1` rather than `vectorMagnitude _canvasVect` may work